try:
    import blargie
except ImportError as e:
    e2 = e

import pdb; pdb.set_trace()
